armi/materials/tests/test_materials.py

```diff
@@ -43,6 +45,11 @@ def test_isPicklable(self):
             self.mat.thermalConductivity(500), mat.thermalConductivity(500)
         )
 
+    def test_density3(self):
+        """Test that all materials produce a zero density from density3"""
```

Comment:
@sombrereau 

The docstring here does not appear to align with what is actually being tested.

Can you clarify which way is meant to be true, i.e. if `density3()` is supposed to be zero or not?